modules/nw-ovn-kubernetes-observability.adoc

```diff
@@ -0,0 +1,170 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-observability_{context}"]
+= Checking OVN-Kubernetes network traffic with OVS sampling using the CLI
+:FeatureName: Checking OVN-Kubernetes network traffic with OVS sampling
+include::snippets/technology-preview.adoc[]
+
+OVN-Kubernetes network traffic can be viewed with OVS sampling via the CLI for the following network APIs: `NetworkPolicy`, `AdminNetworkPolicy`, `BaselineNetworkPolicy`, `UserDefinesdNetwork` isolation, `EgressFirewall`, and multicast ACLs. Scripts for these networking events are found in the `/usr/bin/ovnkube-observ` path of each OVN-Kubernetes node.
+
+Unlike the Network Observability Operator, which is intended for observability and debugging, checking OVN-Kubernetes network traffic with OVS sampling using the CLI is intended to help with packet tracing and debuggability. It can be used while the Network Observability Operator is installed, however that is not a requirement.
+
+Administrators can add the `--add-ovs-collect` option to view network traffic across the node, or pass in additional flags to filter result for specific pods. Additional flags can be found in the "OVN-Kubernetes network traffic with OVS sampling flags" section.
+
+Use the following procedure to view OVN-Kubernetes network traffic using the CLI.
+
+.Prerequisites
+
+* You are logged in to the cluster as a user with `cluster-admin` privileges.
+* You have created a source pod and a destination pod and ran traffic between them.
+* You have created at least one of the following network APIs: `NetworkPolicy`, `AdminNetworkPolicy`, `BaselineNetworkPolicy`, `UserDefinesdNetwork` isolation, multicast, or egress firewalls.
+
+.Procedure
+
+. Edit the `FeatureGate` CR named `cluster` by entering the following command:
```

Comment:
@stevsmit From my test, if cluster is created by techpreview, `OVNObservability` enabled by default, not require STEP 1 and 2. 
When cluster is normal, yes, we need to enable it. 
BTW I think the better way to enable this feature should use `oc patch --type=merge --patch '{"spec": {"featureSet": "TechPreviewNoUpgrade"}}' featuregate/cluster` cc @npinaeva